feature-libs/customer-ticketing/occ/adapters/occ-customer-ticketing.adapter.ts

```diff
@@ -80,7 +85,18 @@ export class OccCustomerTicketingAdapter implements CustomerTicketingAdapter {
     return this.http
       .get<TicketDetails>(this.getTicketEndpoint(customerId, ticketId))
       .pipe(
-        catchError((error) => throwError(normalizeHttpError(error))),
+        catchError((error) => {
+          if(error.error.errors[0].type === 'NotFoundError'
```

Comment:
put this error in a constant

Reply:
Done. I've now added a new module for constants, as no such module existed earlier.